python/lsst/daf/butler/configs/storageClasses.yaml

```diff
@@ -171,3 +171,7 @@ storageClasses:
     pytype: lsst.verify.Measurement
   Stamps:
     pytype: lsst.meas.algorithms.Stamps
+  AstroTable:
```

Comment:
I think I'd prefer AstropyTable since that's what it is.

Reply:
Oh, rats.  I missed that.  Incoming